This patch solves the issues in the build from source from Slackware
It is extracted from the last changes committed in the current git branch 
by the author Team-TL.
Thanks to Team-TL (aka Tomasz Łuczak)

--- fotokilof/fotokilof.sh	2023-04-26 21:00:46.000000000 +0200
+++ fotokilof/fotokilof.sh	2023-06-01 19:05:41.388973243 +0200
@@ -2,5 +2,5 @@
 
 # MYPATH - path to FotoKilof python source
-MAIN=fotokilof.py
+MAIN=__main__.py
 MYPATH="${0%/*}"
 cd $MYPATH

--- ./fotokilof/__init__.py	2023-04-26 21:00:46.000000000 +0200
+++ ./fotokilof/__init__.py	2023-06-01 19:04:14.564973735 +0200
@@ -2,5 +2,5 @@
 
 """
-Copyright (c) 2019-2022 Tomasz Łuczak, TeaM-TL
+Copyright (c) 2019-2023 Tomasz Łuczak, TeaM-TL
 
 Permission is hereby granted, free of charge, to any person obtaining a copy

--- ./setup.py	2023-04-26 21:00:46.000000000 +0200
+++ ./setup.py	2023-06-01 19:01:40.430974610 +0200
@@ -1,3 +1,3 @@
-import setuptools
+from setuptools import setup, find_packages
 
 with open("README.md", "r") as fh:
@@ -7,5 +7,5 @@
 with open("fotokilof/version.py") as fp:
     exec(fp.read(), pkg_vars)
-setuptools.setup(
+setup(
     name=pkg_vars['__appname__'],
     version=pkg_vars['__version__'],
@@ -17,5 +17,4 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
-    packages=setuptools.find_packages(),
     license="MIT",
     classifiers=[
@@ -26,11 +25,12 @@
     ],
     python_requires='>=3.6',
-    install_requires=['pillow','wand','ttkbootstrap'],
-    entry_points = {
-        "gui_scripts": [
-            "fotokilof = fotokilof:__main__",
-        ]
-    },
     include_package_data=True,
+    packages=find_packages(),
+    install_requires=['pillow','wand','ttkbootstrap'],
+#    entry_points = {
+#        "gui_scripts": [
+#            "fotokilof = fotokilof:__main__",
+#        ]
+#    },
 )
 
--- ./scripts/0_pypi_setuptools.sh	2023-04-26 21:00:46.000000000 +0200
+++ ./scripts/0_pypi_setuptools.sh	2023-06-01 19:07:20.691972680 +0200
@@ -4,13 +4,8 @@
 rm -rf ../fotokilof/__pycache__
 CWD=`pwd`
-INITPY="fotokilof/__init__.py"
-MAINPY="fotokilof/fotokilof.py"
-MAIN__PY="fotokilof/__main__.py"
 
 echo "- PyPI ---"
 
 cd ..
-touch $INITPY
-mv $MAINPY $MAIN__PY
 
 # Python 3.8
@@ -19,5 +14,4 @@
 python3 setup.py sdist bdist_wheel
 
-mv $MAIN__PY $MAINPY
 cd $CWD
 
--- ./MANIFEST.in	2023-04-26 21:00:46.000000000 +0200
+++ ./MANIFEST.in	2023-06-01 19:00:41.446974944 +0200
@@ -11,4 +11,4 @@
 
 # Include locale
-recursive-include fotokilof/locale *
+recursive-include locale *.mo
 
